### ISSUE TITLE
color picker for last message line does not work

### ISSUE THREAD

**Issue by M4tthewDE** [Chatterino/chatterino2#2119]
**Describe the bug**
Color picker for "Last message line color" does not work. Stays black after selecting another color. 

**To reproduce**
Change color of "Last message line" in settings to anything besides black. Stays black after confirming new color in color picker. 

**Chatterino version**
Chatterino 2.2.3-beta (commit ec9486948)

**Operating system**
Arch Linux 5.9.1-arch1-1

**Comment by leon-richardt** [comment]
Can reproduce, also on Arch Linux 5.9.1-arch1-1 / Chatterino 2.2.3-beta (commit ec94869).

**Comment by daiexd** [comment]
application now totally crashes when attempting to change the color. Windows 10 / Chatterino Nightly 2.2.3-beta (commit 466d85c90).

**Comment by emoneyLW** [comment]
I've managed to get three different behaviours out of this, so it's not a consistent problem.

In commit 4199a01b9, which I am running as an AppImage downloaded from a nightly I have seen the colour picker both work without any issues but I have also seen it crash when I try press the okay button choosing a colour.

I have also built 4199a01b9 from source and in that instance of the program, using the colour picker will set the colour to black regardless of what was in the settings file prior, but have not managed to get it to crash nor has it ever succeeded.

In both cases, chatterino has been opened from the terminal.
